dataservice/api/common/model.py

```diff
@@ -104,21 +77,36 @@ class IndexdFile:
     # files in indexd cannot be looked up by their baseid
     latest_did = db.Column(UUID(), nullable=False)
 
-    # Fields used by indexd, but not tracked in the database
-    file_name = IndexdField('')
-    urls = IndexdField([])
+    file_name = ''
```

Comment:
Do you still need all these class properties since you changed them now to be instance level properties?

Reply:
Yeah, sqlalchemy uses them to initialize the instance. Otherwise, they do not show up when you construct a new instance.

Reply:
ah ok 👍